packages/_config/src/eslint.js

```diff
@@ -111,10 +111,13 @@ export const createConfig = async () =>
         'no-unused-vars': [
           WARN,
           {
-            args: 'after-used',
-            argsIgnorePattern: '^_',
-            ignoreRestSiblings: true,
-            varsIgnorePattern: '^ignored',
+            "args": "all",
+            "argsIgnorePattern": "^_",
+            "caughtErrors": "all",
+            "caughtErrorsIgnorePattern": "^_",
+            "destructuredArrayIgnorePattern": "^_",
+            "varsIgnorePattern": "^_",
+            "ignoreRestSiblings": true
```

Comment:
These are the actual changes, the other changes are prettier formatting indentation

Reply:
can you talk about these?

Reply:
yes, ended up taking this from here https://johnnyreilly.com/typescript-eslint-no-unused-vars since prefixing unused vars with _ wasn't working